### PR TITLE
Replace Support with AndroidX

### PR DESCRIPTION
**Pull request by I-am-original**
Hi there. Here is update to use AndroidX instead Support lib. 
Please take a look and merge if it is ok for you. 

Cheers

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -22,15 +22,14 @@ rootProject.allprojects {
...
     defaultConfig {
         minSdkVersion 18
-        targetSdkVersion 27
-        versionCode 5
-        versionName "3.1.1"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        targetSdkVersion 28
+        versionCode 6
+        versionName "3.1.4"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```
Comment (mogol) [inline android/build.gradle]: There are no android tests, 
could `testInstrumentationRunner` be deleted?

Comment (I-am-original) [reply android/build.gradle]: @mogol as it was included before - I left it as it is. Maybe you would like to write some test ) at some point of time )

Comment (mogol) [reply android/build.gradle]: Does the plugin need any references to SupportLibrary/AndroidX?
Looks like if I remove any references to tests libs, then SupportLibrary/AndroidX are not needed.

Comment (I-am-original) [reply android/build.gradle]: @mogol  ok you can left it as it is. up to you

**Pull request closed without merging**